docs/dietpi_tools/system_maintenance.md

```diff
@@ -94,18 +106,25 @@ dietpi-backup
     - `.dietpi-backup_stats` gives a list of completed operations with time and date
     - `.dietpi-backup.log` gives a list of every processed file
 
-### Location of the backup
+### Important DietPi-Backup hints
 
-The location of the backup can be set to any directory which is not part of the backup itself. Especially, these options are typical:
+!!! note "File system type of the backup directory: NFS needed!"
 
-- Backup to/Restore from a connected USB stick or USB disk
-- Backup to/Restore from a network drive (Linux file system needed!)
+    DietPi-Backup is purely based on `Rsync`, i.e., DietPi-Backup uses a backup on a file by file basis and is not based on an archive algorithm. This leads to the fact that file permissions/attributes are only preserved correctly in case of a Linux file system as the target directory.   
+    In other cases (e.g. a Samba share), file permissions are not saved in the right manner and DietPi-Backup will not work appropriate.
 
-!!! info "Backup to/Restore from a network drive: Only Linux file systems are usable (NFS)"
+    **Therefore, DietPi-Backup should only be used to/from a directory which handles Linux file attributes (e.g. NFS).**
```

Comment:
Same here, NFS is only a good example if the whole section is clearly about network drives. If it is a general section about filesystem requirements, then probably `ext4` is a better example.

Reply:
Changed the text a bit.